testing/i3wm-gaps/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Jakub Skrzypnik <j.skrzypnik@openmailbox.org>
+pkgname=i3wm-gaps
+pkgver=4.13
+pkgrel=0
+pkgdesc="A tiling window manager with gaps and other additional features"
+url="https://github.com/Airblader/i3"
+arch="all"
+license="BSD"
+replaces="i3wm"
+makedepends="bison flex libxcb-dev xcb-util-cursor-dev
+	xcb-util-keysyms-dev xcb-util-wm-dev libev-dev pango-dev cairo-dev
+	yajl-dev startup-notification-dev pcre-dev libxkbcommon-dev
+	xcb-util-xrm-dev asciidoc perl xmlto autoconf automake"
+subpackages="$pkgname-doc"
+source="i3wm-$pkgver.tar.gz::https://github.com/Airblader/i3/archive/4.13.tar.gz
```

Comment:
Please don’t hard-code version in source URL (it complicates bumping of abuild), use variable `$pkgver`.